src/player.h

```diff
@@ -1853,8 +1855,8 @@ class player : public Character
          * Warmth from terrain, furniture, vehicle furniture and traps.
          * Can be negative.
          **/
-        static int floor_bedding_warmth( const tripoint &pos );
-        /** Warmth from clothing on the floor **/
+        static int floor_bedding_warmth( const tripoint
+                                         &pos );        /** Warmth from clothing on the floor **/
```

Comment:
looks like a typo caused this comment to be on the same line here. you should change this back to how it was.

Reply:
...I'm not even sure what caused that one.

Reply:
Upon further investigation I found out that astyle really doesn't like how long that line is.